docs/_data/menu-sql.yaml

```diff
@@ -0,0 +1,81 @@
+- text: Getting Started
+  url: sql-getting-started.html
+  subitems:
+    - text: "Starting Point: SparkSession"
+      url: sql-getting-started.html#starting-point-sparksession
+    - text: Creating DataFrames
+      url: sql-getting-started.html#creating-dataframes
+    - text: Untyped Dataset Operations (DataFrame operations)
+      url: sql-getting-started.html#untyped-dataset-operations-aka-dataframe-operations
+    - text: Running SQL Queries Programmatically
+      url: sql-getting-started.html#running-sql-queries-programmatically
+    - text: Global Temporary View
+      url: sql-getting-started.html#global-temporary-view
+    - text: Creating Datasets
+      url: sql-getting-started.html#creating-datasets
+    - text: Interoperating with RDDs
+      url: sql-getting-started.html#interoperating-with-rdds
+    - text: Aggregations
+      url: sql-getting-started.html#aggregations
+- text: Data Sources
+  url: sql-data-sources.html
+  subitems:
+    - text: "Generic Load/Save Functions"
+      url: sql-data-sources-load-save-functions.html
+    - text: Parquet Files
+      url: sql-data-sources-parquet.html
+    - text: ORC Files
+      url: sql-data-sources-other.html#orc-files
+    - text: JSON Datasets
+      url: sql-data-sources-other.html#json-datasets
+    - text: Hive Tables
+      url: sql-data-sources-hive-tables.html
+    - text: JDBC To Other Databases
+      url: sql-data-sources-jdbc.html
+    - text: Avro Files
+      url: sql-data-sources-avro.html
+    - text: Troubleshooting
+      url: sql-data-sources-other.html#troubleshooting
```

Comment:
Hi, @xuanyuanking . Generally, it looks good.

Can we split `sql-data-sources-other` into three files? For me, `troubleshooting` looks weird in terms of level of information. Actually, `sql-data-sources-other` has only two files and `troubleshooting` for JDBC.

Maybe, `sql-data-sources-orc`, `sql-data-sources-json` and `toubleshooting`?

Reply:
Make sense, will split into `sql-data-sources-orc`, `sql-data-sources-json` and `sql-data-sources-troubleshooting`(still need sql-data-sources prefix cause [here](https://github.com/apache/spark/pull/22746/files#diff-5075091c2498292f7afcac68bfd63e1eR13) we need "sql-data-sources" as the nav-left tag, otherwise the nav menu will not show the subitems).